src/components/ArtifactSigner/artifact-signer.tsx

```diff
@@ -0,0 +1,108 @@
+import './artifact-signer.scss';
+import {
+  Button,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExternalLinkSquareAltIcon } from '@patternfly/react-icons';
+import React from 'react';
+import { ExternalLink } from '../ExternalLink';
+
+type ArtifactSignerProps = {
+  currentStep: number;
+  onDownloadSecret: () => void;
+  onDocumentationClick: () => void;
+  userHasDownloadedSecret: boolean;
+  nextStep: () => void;
+  previousStep: () => void;
+};
+
+const GettingStarted = () => {
+  return (
+    <div>
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Trusted Artifact Signer empowers software developers and consumers to
+          securely sign and verify software artifacts - source code, release
+          files, images, binaries, SBOM, playbook, and more to enhance software
+          supply chain security. Signing metadata are stored in a
```

Comment:
Suggesting a minor edit for grammar, and style for this first sentence.

`... verify software artifacts - source code, release files, images, binaries, SBOM, playbook ...`  --> 

`... verify software artifacts, such as, source code, release files, container images, binaries, software bill of materials (SBOM), playbooks ...`